postinstall

```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
 
+return_code=0
```

Comment:
Does the failure produce a human-understandable error?

I checked and a think they do, but please revise them for grammar (most of them have issues), also use the same formatting we usually do (lower-case initial character for error messages as well as log messages).

Reply:
yes, it produces an error message like following:
```
Error: Failed to locate a version of chromedriver that matches the installed version of chrome (99). 
Valid chrome versions are: 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73
    at driverInstaller 
        ...
        throw new Error(
        ^

Error: Failed to locate a version of geckodriver that matches the installed version of firefox (88).
Valid firefox versions are: 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66 
 ...
```